contrib/Detached.Mappers.EntityFramework.Contrib.SysTec/ComplexDeepModelBugReproTests.cs

```diff
@@ -171,7 +171,7 @@ public void _03_RemoveChangeAndAddEntriesInTagListComposition_ShouldBeSupportedI
             _superCustomer.Tags = new List<Tag>
             {
                 // Tag1 removed - will not be sent back by client
-                new Tag() { Id = _tag2.Id, Name = "Changed Marketing Campaign1" },
+                new Tag() { Id = _tag2.Id, Name = "Changed Marketing Campaign1", ConcurrencyToken = 2 },
```

Comment:
here if the Tag entity is used, then ConcurrencyToken must be set, otherwise, it is 0 and the update failed
to avoid this, a TagDTO or an anonymous may be used
(this is why I always recommend to avoid using entities as dtos, there is always fields that should not be mapped, like passwords, create/update time, and this one)

Reply:
I don't really understand. Because the Tag is saved in DB, it should has the ConcurrencyToken = 1. In this line it will be changed, so it should have value still 1, because it is the original version to change.

But I agree that DTO would be better - here is a good documentation, why! :-)